tests/test_beatles.py

```diff
@@ -21,27 +21,23 @@ def data_home(tmpdir):
 
 
 @pytest.fixture
-def save_path(data_home):
-    return utils.get_save_path(data_home)
+def dataset_path():
+    return os.path.join(beatles.DATASET_DIR)
 
 
-@pytest.fixture
-def dataset_path(save_path):
-    return os.path.join(save_path, beatles.DATASET_DIR)
-
-
-@pytest.fixture
-def mock_beatles_exists(mocker):
-    return mocker.patch.object(beatles, 'exists')
```

Comment:
I _think_ if you do `return mocker.patch.object(os.path, 'exists')`, this might "just work", as we say in the business;

You might need to do `return mocker.patch.object('os.path', 'exists')`, though - Python mocks are particular.